netbox/templates/extras/script_list.html

```diff
@@ -22,23 +22,28 @@
   {% for module in script_modules %}
     {% include 'inc/sync_warning.html' with object=module %}
     <div class="card">
-      <h2 class="card-header justify-content-between" id="module{{ module.pk }}">
-        <div>
-          <i class="mdi mdi-file-document-outline"></i> {{ module }}
+
+      <div class="card-header container-fluid">
+        <div class="col col-md-10">
+          <h2 id="module{{ module.pk }}">
+            <div>
+              <i class="mdi mdi-file-document-outline"></i> {{ module }}
+            </div>
+          </h2>
         </div>
-        <div>
-          {% if perms.extras.edit_scriptmodule %}
-            <a href="{% url 'extras:scriptmodule_edit' pk=module.pk %}" class="btn btn-warning btn-sm">
-              <i class="mdi mdi-pencil" aria-hidden="true"></i> {% trans "Edit" %}
-            </a>
-          {% endif %}
-          {% if perms.extras.delete_scriptmodule %}
-            <a href="{% url 'extras:scriptmodule_delete' pk=module.pk %}" class="btn btn-danger btn-sm">
-              <i class="mdi mdi-trash-can-outline" aria-hidden="true"></i> {% trans "Delete" %}
-            </a>
-          {% endif %}
+        <div class="col col-md-2 d-print-none">
+          <div class="btn-list justify-content-end mb-2">
+            {% if perms.extras.edit_scriptmodule %}
+              {% url 'extras:scriptmodule_edit' pk=module.pk as edit_url %}
+              {% include 'buttons/edit.html' with url=edit_url %}
```

Comment:
These templates exist solely for the button template tags; we should avoid including them directly.